firefox_puppeteer/ui/windows.py

```diff
@@ -188,7 +188,7 @@ def __eq__(self, other):
 
     @property
     def closed(self):
-        """Returns closed state of the chrome window
+        """Returns closed state of the chrome window.
```

Comment:
For some you removed the dot, and here you add it. I would suggest we always keep it.